AdyenCashAppPay/CashAppPayComponent.swift

```diff
@@ -23,6 +23,12 @@ public final class CashAppPayComponent: PaymentComponent,
         static let storeDetailsItem = "storeDetailsItem"
         static let cashAppButtonItem = "cashAppButtonItem"
     }
+    
+    private enum ErrorMessage {
+        static let unexpectedError = "CashApp unexpected error"
```

Comment:
Do we have an agreement with data team to send human readable values? Is it the same for Android and Web?

Reply:
The message field is only for context, not to be used as a filtering property so each platform is free to send any logical, non sensitive text.

Reply:
Why do we have such parameter then? What's the benefit from a product or data point?

Reply:
Should have also added that message field is only sent when the same error code event is sent from multiple places. Just an extra help when we filter by for example thirdParty errors for component X, we will see how many unique cases there are.